### ISSUE TITLE
Reddit secret key shared publicly

### ISSUE THREAD

**Issue by mohataher** [tsCoelho/BuzzDetector#1]
Shouldn't [this](https://github.com/tsCoelho/BuzzDetector/blob/master/reddit/reddit_get_info.py#L6) be stored in untracked file?

**Comment by tsCoelho** [comment]
Yes, you are absolutely right. Thanks for the heads up! I'll make some changes on the visibility of credentials; wasn't expecting the repo to get spotted in this short time.
